Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -69,7 +69,7 @@ public struct WebAuthnManager: Sendable {
             user: user,
             relyingParty: .init(id: configuration.relyingPartyID, name: configuration.relyingPartyName),
             publicKeyCredentialParameters: publicKeyCredentialParameters,
-            timeout: timeout,
+            timeout: (timeout?.milliseconds ?? 0) / 1000,
```

Comment:
This should still be unnecessary?

Reply:
Yes, updated in commit `ee000536baa6801d50b9c4a1242f43a9be1d71f1`